java/semantickernel-api/src/main/java/com/microsoft/semantickernel/Kernel.java

```diff
@@ -129,6 +130,7 @@ class Builder {
         @Nullable private PromptTemplateEngine promptTemplateEngine = null;
 
         @Nullable private ReadOnlySkillCollection skillCollection = null;
+        @Nullable private MemoryStore memoryStore = null;
```

Comment:
To mirror c#, the memory should be a SemanticTextMemory, and should probably default to NullMemory.